tpm/tpm.go

```diff
@@ -754,6 +754,11 @@ func ReadPubEK(rw io.ReadWriter) ([]byte, error) {
 	return tpmutil.Pack(pk)
 }
 
+// GetManufacturer returns the manufacturer ID
+func GetManufacturer(rw io.ReadWriter) ([]byte, error) {
+	return getCapability(rw, capProperty, tpmCapPropManufacturer)
```

Comment:
nit: you probably don't need getCapability as a different method if its only used here.

Reply:
Although it is only used here it can be used to get a variety of other fields so I figured it would be best to abstract now.

Reply:
Can you re-use this for GetKeys then?

Reply:
Done